variables.tf

```diff
@@ -204,3 +204,21 @@ variable "ca_cert_identifier" {
   description = "The identifier of the CA certificate for the DB instance"
   default     = null
 }
+
+variable "ssm_parameter_enabled" {
+  type        = bool
+  default     = false
+  description = "Whether an SSM parameter store value is created to store the key's private key pem."
+}
+
+variable "ssm_parameter_type" {
+  type        = string
+  default     = "SecureString"
+  description = "Type of the ssm parameter. Valid types are String, StringList and SecureString."
+}
```

Comment:
Probably doesn't need to be a variable, it should not be allowed to store as `String`

Reply:
okay, will store it as `SecureString` for the default on this resource. thank you @kevcube